libs/design-system/content-templates/api-command/src/lib/api-command.schema.ts

```diff
@@ -0,0 +1,31 @@
+import { ContentTemplateSchema } from '@hra-ui/cdk/content-template';
+import { z } from 'zod';
+
+/** API command button type */
+export type APICommandButton = z.infer<typeof APICommandButtonSchema>;
+
+/** Schema representing the details of an api command button */
+export const APICommandButtonSchema = z.object({
+  /** Text for the plain button */
+  label: z.string(),
+  /** Icon name for the button */
+  icon: z.string().optional(),
+  /** External URL for the button */
+  url: z.string().url().optional(),
+});
+
+/** API command type */
+export type APICommand = z.infer<typeof APICommandSchema>;
+
+/** Schema for api command */
+export const APICommandSchema = ContentTemplateSchema.extend({
+  component: z.literal('APICommand'),
+  /** API url to be displayed in the card */
+  url: z.string(),
+  /** Can be GET or POST */
+  function: z.string(),
+  /** Details of the plain button */
+  leftButton: APICommandButtonSchema,
```

Comment:
The left button is always `Copy` so there is no need to specify it dynamically. As for the right button I'm not sure. Please check with Libby whether it will always be `Try me` or if it can be something else